types/src/legacy_sse_data/translate_execution_result.rs

```diff
@@ -201,7 +209,88 @@ mod tests {
             Some(TransformKindV1::WriteContractPackage),
             maybe_tanslate_stored_value(&stored_value)
         );
-        //TODO wrtite tests for rest of cases
+    }
+
+    #[test]
+    fn default_execution_effects_translator_should_translate_effects_v2() {
+        let mut rng = TestRng::new();
+        let under_test = DefaultExecutionEffectsTranslator {};
+        let key_1: Key = rng.gen();
+        let key_2: Key = rng.gen();
+        let key_3: Key = rng.gen();
+        let effects = build_example_effects(key_1, key_2, key_3);
+
+        let maybe_translated = under_test.translate(&effects);
+
+        assert!(maybe_translated.is_some());
+        let translated = maybe_translated.unwrap();
+        assert!(translated.operations.is_empty());
+        assert_eq!(
+            translated.transforms,
+            build_expected_transforms(key_1, key_2, key_3)
+        );
+    }
+
+    #[test]
+    fn default_execution_effects_translator_should_empty_trnsforms_if_something_was_not_translatable(
+    ) {
+        let mut rng = TestRng::new();
+        let under_test = DefaultExecutionEffectsTranslator {};
+        let key_1: Key = rng.gen();
+        let key_2: Key = rng.gen();
+        let key_3: Key = rng.gen();
+        let mut effects = build_example_effects(key_1, key_2, key_3);
+        effects.push(TransformV2::new(
+            Key::Account(rng.gen()),
+            TransformKindV2::Write(StoredValue::Message(MessageChecksum([1; 32]))),
+        ));
+
+        let maybe_translated = under_test.translate(&effects);
+
+        assert!(maybe_translated.is_some());
```

Comment:
```suggestion
        assert!(maybe_translated.is_some(), "{:?}", maybe_translated);
```
for asserts like this I like to add a message with context, otherwise it's just going to fail with 'expected true but got false'

Reply:
:+1: